### DISCUSSION TITLE
code block: is it possible to enable copy, without line numbers?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#891] [Q&A] [answered]
Hi,
if I use simply

~~~
```
sed -r 's/^.+=//g;s/,\];/]/g'
```
~~~

I have no line numbers and no copy block.

![image](https://user-images.githubusercontent.com/30607/168557789-4c28258d-7c27-47e0-a311-afa55219aded.png)

If I add `{.txt}`, I have both line numbers and copy button. 

~~~
```{.txt}
sed -r 's/^.+=//g;s/,\];/]/g'
```
~~~

![image](https://user-images.githubusercontent.com/30607/168558159-4ee4ca62-9f2f-441d-b1cd-1a2f7b86e35c.png)

Is there a way to have copy button, and no line numbers?

I have tried `{.txt line_numbers=false}`, but I have always the line number.

Thank you

**Comment by jjallaire** [comment]
Yes, try this:

```yaml
---
format: 
  revealjs:
    code-line-numbers: false
    code-copy: false
---
```

**Reply by aborruso** [reply]
Thank you @jjallaire, this is not what I want. I explained myself badly, sorry

My goal is to have by default line numbers and code-copy, and to be able to choose to have in example only in one block code the code copy, but not the line number.

**Comment by jjallaire** [comment, marked as answer]
You can do this:

````markdown
```{.txt code-line-numbers="false"}
sed -r 's/^.+=//g;s/,\];/]/g'
```
````
